bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/DocsCommand.java

```diff
@@ -15,16 +17,18 @@
 import net.dv8tion.jda.api.entities.User;
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
 
-import java.awt.Color;
+import java.awt.*;
+import java.util.List;
 import java.util.*;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 
 public class DocsCommand extends ReactionCommand
 {
     private static final int RESULTS_PER_PAGE = 5;
     private static final String[] ALIASES = new String[]{"documentation", "doc", "jdoc", "jdocs"};
+    private final ButtonListener buttons = new ButtonListener();
+    private boolean first = true;
```

Comment:
Not really a fan of this handling. would probably be better to simply create + register the listener in the Dispatcher or Main class and pass it into the DocsCommand through constructor or through some getter on eg the dispatcher